data/json/flags.json

```diff
@@ -1170,6 +1170,12 @@
     "//": "this is deprecated for the COLLAPSED_STOCK and FOLDED_STOCK, kept around for mod support",
     "type": "json_flag"
   },
+  {
+    "id": "REMOVED_STOCK",
+    "//": "for weapons that the stock has been cut off.",
+    "type": "json_flag",
+    "info": "This items has had its stock removed it will be hard to shoot with."
```

Comment:
```suggestion
    "//": "for weapons whose stock has been cut off.",
    "type": "json_flag",
    "info": "This item has had its stock removed, it will be hard to shoot with."
```